DigitalSignageUAP/SlideshowPage.xaml.cs

```diff
@@ -240,11 +256,15 @@ void PlayAudio()
 
```

Comment:
No, that is not necessary/desired.
We can have image slideshow and audio playback going on simultaneously. On the other hand, when video is playing, then we collapse/pause audio playback and image slideshow.